src/cljs/main/broadfcui/page/workspace/data/tab.cljs

```diff
@@ -131,13 +132,9 @@
                  :value (utils/get-access-token)}]
         [:input {:type "hidden"
                  :name "attributeNames"
-                 :value (->> (persistence/try-restore
-                              {:key (str (common/workspace-id->string
-                                          workspace-id) ":data:" selected-entity-type)
-                               :initial (constantly {})})
-                             :column-meta
+                 :value (->> (or (:visible-columns @state) ((@refs "entity-table") :get-ordered-columns))
                              (filter :visible?)
-                             (map :header)
+                             (map (some-fn :id :header))
```

Comment:
I was wrong--this should just be `(map :id)`. The table uses the `:id` property, and can use `:header` if it's a string and `:id` isn't provided. But by this point, it's resolved that and just refers to columns by `:id`.